src/screens/Cart/index.tsx

```diff
@@ -1,10 +1,59 @@
-import { Box,Text } from "native-base";
+import { AddIcon, Box, Button, ChevronLeftIcon, Heading, HStack, Image, MinusIcon, Stack, Text, VStack } from "native-base";
+import { useState } from "react";
 
-export default function Card() {
+export default function Cart() {
+    const [quant, setQuant] = useState(1)
+    const [price, setPrice] = useState(2.03)
+    const [bDisable, setBDisable] = useState(false)
 
-    return(
-        <Box>
-            <Text>Card</Text>
-        </Box>
+    console.log(quant)
+    return (
+        <>
+            <Box>
+                <ChevronLeftIcon size="5" />
+            </Box>
+            <Heading fontSize="40">Cart</Heading>
+            <Box alignSelf="center">
+
+
+                <Box flexDirection="row" rounded="lg" overflow="hidden" borderWidth={2} borderColor="gray.200">
+
+
+                    <Image
+                        source={{
+                            uri: "https://wallpaperaccess.com/full/317501.jpg"
+                        }}
+                        alt="Alternate Text"
+                        size={120}
+                    />
+                    <VStack>
+                        <Box paddingX={4} paddingY={4}>
+                            <Heading size="md">Carlsberg Premium Beer</Heading>
+                            <Text fontSize="xs">375ml Can  |  5%</Text>
+                        </Box>
+
+                        <HStack>
+                            <Stack space="2.5" px="8">
+                                <Stack direction="row" mb="2.5" mt="1.5" space={3}>
+                                    <Button size="6" variant="subtle" onPress={() => quant <= 1 ? setBDisable(true) : setQuant(quant - 1)} isDisabled={bDisable}>
+                                        <MinusIcon />
+                                    </Button>
+                                    <Text fontSize="md">{quant}</Text>
+                                    <Button size="6" variant="subtle" onPress={() => {
```

Comment:
Nunca é bom passarmos funções anônimas como props assim, devemos sempre criar uma função no escopo do componente pra isso, mesmo que pareça redundante
```suggestion
                                    <Button size="6" variant="subtle" onPress={handleAddItemToCart}
```

Reply:
Ahh, o motivo, é que sempre que a página é  re-renderizada ela declara a função novamente, usando mais memória assim